esphome/components/mhz19/sensor.py

```diff
@@ -2,17 +2,19 @@
 import esphome.config_validation as cv
 from esphome.components import sensor, uart
 from esphome.const import CONF_CO2, CONF_ID, CONF_TEMPERATURE, ICON_PERIODIC_TABLE_CO2, \
-    UNIT_PARTS_PER_MILLION, UNIT_CELSIUS, ICON_THERMOMETER
+    UNIT_PARTS_PER_MILLION, UNIT_CELSIUS, ICON_THERMOMETER, UNIT_EMPTY, ICON_EMPTY
 
 DEPENDENCIES = ['uart']
 
+CONF_ABC = "automatic_baseline_calibration"
```

Comment:
```suggestion
CONF_AUTOMATIC_BASELINE_CALIBRATION = "automatic_baseline_calibration"
```

CONF_ name should match option name